### DISCUSSION TITLE
Some links do not render in recent wiki.js versions

### DISCUSSION THREAD

**Discussion by talalvas** [requarks/wiki#3033] [Help / Questions]
**Describe the bug**
_Hello dear forum users and contributors._
I'm using hyper link that activates an internal application that runs on my IIS server.
The problem is when i try to create links of this kind, the system don't render them into hyper links in the output rendered page.
This problem started on one of the recent versions, unfortunatly i can't put my finger exactly on which version.
I've double checked this issue by making a copy of an old page created a while ago in which the links works.
In the duplicated page the links don't work (see attachments)

the link look like this:
`priority:priform@DOCUMENTS_D::.:tabula.ini`
it opens a new window in my ERP software.
When i click the link on the old page - it works.
When i put the link directly in the browser address bar - it works.

When you inspec the link element in the new version you will get:
`<a class="is-asset-link">משלוחים ללקוח</a>`
In the old version you will get:
`<a href="priority:priform@DOCUMENTS_D::.:tabula.ini" class="is-asset-link">משלוחים ללקוח</a>`

I guess something prevent from links such as this to render in later wiki.js versions.

**To Reproduce**
1. Create new page
2. Create a link in that page with this code: [Test This Link](priority:priform@DOCUMENTS_D::.:tabula.ini)
3. Create and display the page
4. Try pressing the link

**Expected behavior**
You would expect that the link will be clickable like in previous versions.

**Screenshots**
![ss2](https://user-images.githubusercontent.com/9365114/106882359-54232500-66e7-11eb-9ed1-7d0c71041b26.png)
**Link in the new version not working**

![ss1](https://user-images.githubusercontent.com/9365114/106882253-381f8380-66e7-11eb-905a-d46531c1bd37.png)
**Link in the older version is working**

![ss3](https://user-images.githubusercontent.com/9365114/106882434-69984f00-66e7-11eb-9574-bd339c15050a.png)
**In the older version page you can see the link when you hover on the link text. in the new version nothing shows when you hover on the link text.**

**Host Info:**
 - OS: Windows Server 2019 Standard (Windows_NT (win32) 10.0.17763 x64)
 - Wiki.js version: 2.5.170
 - Database engine: PostgreSQL 12.1

**Comment by NGPixel** [comment]
You need to disable **Sanitization** under Admin > Rendering > HTML > Security. Then save your page again.

Your links are detected as potentially unsafe which is why they are stripped.
